zone/lua_general.cpp

```diff
@@ -1732,6 +1732,7 @@ void lua_add_spawn_point(luabind::adl::object table) {
 		uint32 variance;
 		uint32 timeleft = 0;
 		uint32 grid = 0;
+		bool path_when_zone_idle;
```

Comment:
Can probably initialize as false, not sure if necessary.

Reply:
Fixing.